### PR TITLE
isolib: fix rt column reading and writing

### LINKED ISSUE #524: isolib: fix rt column reading and writing

### PR DESCRIPTION
**Pull request by hechth**
Fixes #524

### CODE REVIEW

File: tools/isolib/isolib.R [hunk 1]
```diff
@@ -48,7 +48,7 @@ main <- function() {
     )
 
     if ("rt" %in% colnames(compound_table)) {
-      spectra_df$retention_time <- compound_table$rt
+      spectra_df$retention_time <- as.numeric(sub(",", ".", compound_table$rt, fixed = TRUE))
```
Comment (bgruening) [inline tools/isolib/isolib.R:51]: is this really how you do it in R? Can you not set a locale setting somewhere? So that R respects . vs ,?

Comment (hechth) [reply tools/isolib/isolib.R:51]: We can add a dependency to readr for example to include a more sophisticated table reading function, sure

Comment (bgruening) [reply tools/isolib/isolib.R:51]: I'm just surprised. Those things normally depend on your `locale` setting of your environment.

Comment (hechth) [reply tools/isolib/isolib.R:51]: yeah - but how to tell it to accept both?

Comment (hechth) [reply tools/isolib/isolib.R:51]: how does usegalaxy.eu and usegalaxy.org handle these interoperabilities? :D

Comment (bgruening) [reply tools/isolib/isolib.R:51]: We do set one `locale` for all tools. US I think. So that all tools are compatible ...

**Pull request merged**